nameko/messaging.py

```diff
@@ -388,34 +388,6 @@ def on_consume_ready(self, connection, channel, consumers, **kwargs):
             else:
                 callback()
 
-    def consume(self, limit=None, timeout=None, safety_interval=0.1, **kwargs):
-        """ Lifted from Kombu.
-
-        We switch the order of the `break` and `self.on_iteration()` to
```

Comment:
The default timeout is 0.1 seconds so I don't think there's anything to worry about there. If `timeout` is never set as per the comment it has no effect anyway.

@kollhof can you remember why you did this?